src/services/api.service.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+class ApiService{
+    
+    constructor() {    }
+
+    get(url, headers){
```

Comment:
We should generate the headers inside this service instead of getting it as a parameter.
Please move the method
    getHeaders()
From base service to api.service